### ISSUE TITLE
1.5.x branch invalid

### ISSUE THREAD

**Issue by vomikan** [moneymanagerex/moneymanagerex#3253]
MMEX version:
 - [x] 1.5.1

It use "themes" repo and include themes.cpp
But it should be equalent 1.5.0 release. 
It seems the branch 1.5.x became a clone of master branch.

**Comment by whalley** [comment]
@vomikan Eeek!   Don't know what happened there... Too much Vodka I think, need to stick to coffee!

Think the best way out of this is to either reset the head on the branch to 92445d2bd83d17ef8ae39c549590a3e7ead2b8d7, i.e. invoke this on the v1.5.0 branch

`git reset --hard HEAD~2`

 or re-branch from the prior commit 
92445d2bd83d17ef8ae39c549590a3e7ead2b8d7 (in either case we would need to reapply fbdd7943592f5202b2032497ea58c7bcb0b75ddd).

Not sure a revert will work well.

What do you think? I'm a bit nervous (not a git expert), are you happy to give this a go?

I'll go and hang my head in shame now!

**Comment by vomikan** [comment]
I'm also a bit of a non-guru in git. But, I think, nothing difficult to reset and roll the changes.

**Comment by whalley** [comment]
> PS I'm surprised about vodka, because I prefer whiskey. ))

There was me thinking vodka would relate better (yes, I'm a stupid Brit assuming all Russians prefer vodka!).... I'm quite partial to a single malt myself :-)

**Comment by vomikan** [comment]
```
git switch v1.5.x
git reset --hard 92445d2bd83d17ef8ae39c549590a3e7ead2b8d7
git commit -m "Reverting to the state of the branch 1.5.x to 92445d2"
git push origin +v1.5.x
```

**Comment by whalley** [comment]
Have added to my git cheat sheet!

**Comment by vomikan** [comment]
git reset --hard b26ac4a068b5574e28f7862511a7af595945158e

**Comment by vomikan** [comment]
@whalley 
I am not sure .... It should be OK now

**Comment by whalley** [comment]
Yes, thanks again.

**Comment by vomikan** [comment]
![image](https://user-images.githubusercontent.com/6836805/115866593-8f113b00-a442-11eb-9eb3-598ed4ac0a89.png)

**Comment by whalley** [comment]
We can leave it there... Can always disable menu item if we feel it has issues.

**Comment by vomikan** [comment]
>Do we want to re-assess what we have on master vs v1.5.x branch?

I don'tunderstend.

**Comment by whalley** [comment]
Apologies, should have been clearer.

What I mean is that I had expected the v1.5.x branch to be used for 'priority' bug fixes, i.e. issues that were causing major user issues and 'show stoppers' and would need quick fixes and a fast release.

However we seem to have made quite a lot of changes on the v1.5.x branch that I would not categorize as major.

I was just thinking that perhaps we just roll forward with 1.6 and just create a branch and quick fix release as needed rather than maintaining the two -- I only ever really test the master release and use this as my daily app.

Anyway, just my thoughts, you have more experience on the MMEX user expectations around this. Just that I'm more in favour of taking the latest releases to the wider audience.

**Comment by vomikan** [comment]
I tried to make only critical fixes in the 1.5.x branch. However, some features were poorly migrated since the rejection of 1.4.0. User data and columns to them, budget settings, and so on. I had to make more corrections than expected.
However, the master branch has not gone far in development, so you can release 1.5.1 and continue development in the master branch. I'll fix 3264 bug then release 1.5.1 this weekend.

**Comment by whalley** [comment]
Understood, great work on fixing all those issues.
